### PR TITLE
Prepare for Netlify Deployment

### PR DESCRIPTION
**Pull request by jaronheard**
This PR sets up the CIVIC repo for deployment via Netlify. I've tested the Netlify deployment on [my fork](https://github.com/jaronheard/civic/).

There are four main changes:
- Add build-netlify script -- this has to use yarn workspaces instead of lerna to be compatible with Netlify
- Change http:// urls across the repo to https:// for API compatibility
- Add HTMLWebpackPlugin to generate an index file
- Add _redirects file for Netlify

### CODE REVIEW

File: packages/component-library/src/FooterNew/FooterNew.js [hunk 1]
```diff
@@ -98,6 +98,9 @@ const Footer = props => (
       </div>
       <Logo css={logoStyle(props)} type="squareLogo" />
     </div>
+    <a href="https://www.netlify.com">
+      <img src="https://www.netlify.com/img/global/badges/netlify-color-bg.svg"/>
+    </a>
```
Comment (DingoEatingFuzz) [inline packages/component-library/src/FooterNew/FooterNew.js]: Is this required with the nonprofit package?

Comment (jaronheard) [reply packages/component-library/src/FooterNew/FooterNew.js]: There's a requirement to have a link for the open-source plan (https://www.netlify.com/legal/open-source-policy/). I'm going to tone it down a bit, though

Comment (jaronheard) [reply packages/component-library/src/FooterNew/FooterNew.js]: Actually, removed this for now.

**Comment by jaronheard** [comment]
@DingoEatingFuzz For now, the extent of this change is to allow for deploy previews. This would allow for us to transition away from ECS deployment, but you had identified a few key points around parity, security, and ✨the future✨ that were important considerations before committing to a full transition.

**Comment by DingoEatingFuzz** [comment]
👍 Awesome, thank you for clarifying!

**Pull request merged**